### DISCUSSION TITLE
Render different part of pdf into 1 pdf

### DISCUSSION THREAD

**Discussion by cupskrrtt** [wojtekmaj/react-pdf#1372] [Q&A] [answered]
hey, can anyone help me i'm trying to break my pdf into a different part like in the pseudo-code, anyone have an answer to this?
 `<Document>`
    `<Section1/>`
    `<Section2/>`
 `</Document>`

thanks in advance!

**Comment by wojtekmaj** [comment, marked as answer]
It looks like you confused React-PDF with [@react-pdf/renderer](https://github.com/diegomura/react-pdf). Please file an issue in their repo instead.
